backend/api.py

```diff
@@ -1,12 +1,54 @@
 # create our little application :)
-from flask import Flask
+import configparser
+from flask import Flask, request
+import tweepy
 
 app = Flask(__name__)
 
+def are_you_the_keymaster():
```

Comment:
This may be something we want to call once per instance and then reference on every request.